float64                         latitude
float64                         longitude
---
geometry_msgs::PoseStamped      pose
